pkg/core/table/cpin.go

```diff
@@ -0,0 +1,114 @@
+// // Copyright (c) 2021 by library authors. All rights reserved.
+// // Use of this source code is governed by a BSD-style
+// // license that can be found in the LICENSE file.
+
+// // Implements TCG Storage Core Table operations on Locking SP tables
+
+package table
+
+import (
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core"
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core/method"
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core/uid"
+)
+
+type CPINInfoRow struct {
+	// 5.3.2.12.1 This is the unique identifier of this row in the C_PIN table.
+	UID uid.RowUID
+	// 5.3.2.12.2 This is the name of the C_PIN object.
+	// For C_PIN objects that exist at issuance, this column SHALL NOT be modifiable by the host.
+	Name *string
+	// 5.3.2.12.3 This is a name that MAY be shared by multiple C_PIN objects.
+	// For C_PIN objects that exist at issuance, this column SHALL NOT be modifiable by the host.
+	CommonName *string
+	// 5.3.2.12.4 This is the bytes value to which authentication attempts on this C_PIN object are matched.
+	Password []byte
```

Comment:
@yogev-lb : why `Password`? i see that the convention in the library is to call the struct fields by the exact column names (presumably for ease of cross-referencing the code against the spec). this column is actually called `PIN` in the spec.

Reply:
changed to PIN